openshift/olm/knative-eventing.catalogsource.yaml

```diff
@@ -0,0 +1,4776 @@
+      kind: CustomResourceDefinition
+      metadata:
+        name: routes.serving.knative.dev
+      spec:
+        additionalPrinterColumns:
+        - JSONPath: .status.domain
+          name: Domain
+          type: string
+        - JSONPath: .status.conditions[?(@.type=="Ready")].status
+          name: Ready
+          type: string
+        - JSONPath: .status.conditions[?(@.type=="Ready")].reason
+          name: Reason
+          type: string
+        group: serving.knative.dev
+        names:
+          categories:
+          - all
+          - knative
+          - serving
+          kind: Route
+          plural: routes
+          shortNames:
+          - rt
+          singular: route
+        scope: Namespaced
+        subresources:
+          status: {}
+        version: v1alpha1
+
+    - apiVersion: apiextensions.k8s.io/v1beta1
+      kind: CustomResourceDefinition
+      metadata:
+        name: services.serving.knative.dev
+      spec:
+        additionalPrinterColumns:
+        - JSONPath: .status.domain
+          name: Domain
+          type: string
+        - JSONPath: .status.latestCreatedRevisionName
+          name: LatestCreated
+          type: string
+        - JSONPath: .status.latestReadyRevisionName
+          name: LatestReady
+          type: string
+        - JSONPath: .status.conditions[?(@.type=="Ready")].status
+          name: Ready
+          type: string
+        - JSONPath: .status.conditions[?(@.type=="Ready")].reason
+          name: Reason
+          type: string
+        group: serving.knative.dev
+        names:
+          categories:
+          - all
+          - knative
+          - serving
+          kind: Service
+          plural: services
+          shortNames:
+          - kservice
+          - ksvc
+          singular: service
+        scope: Namespaced
+        subresources:
+          status: {}
+        version: v1alpha1
+
+    - apiVersion: apiextensions.k8s.io/v1beta1
+      kind: CustomResourceDefinition
+      metadata:
+        name: installs.serving.knative.dev
+      spec:
+        group: serving.knative.dev
+        names:
+          kind: Install
+          listKind: InstallList
+          plural: installs
+          singular: install
+          shortNames:
+          - ksi
+        scope: Namespaced
+        subresources:
+          status: {}
+        validation:
+          openAPIV3Schema:
+            properties:
+              apiVersion:
+                type: string
+              kind:
+                type: string
+              metadata:
+                type: object
+              spec:
+                type: object
+              status:
+                type: object
+        version: v1alpha1
+        versions:
+        - name: v1alpha1
+          served: true
+          storage: true
+  clusterServiceVersions: |-
+    - apiVersion: operators.coreos.com/v1alpha1
+      kind: ClusterServiceVersion
+      metadata:
+        name: knative-build.v0.1.2
```

Comment:
My instructions were obviously not clear enough, because we don't want `knative-build.v0.1.2` in your CatalogSource. To prevent that, you need to move the files related to the latest eventing release out into their own directory *before* running the `catalog.sh` script. Make sense?